system-tests/project-fixtures/react/src/Unmount.cy.jsx

```diff
@@ -17,10 +17,10 @@ describe('Comp with componentWillUnmount', () => {
     cy.contains('My component')
 
     // after we have confirmed the component exists let's remove it
-    // unmount() command is automatically enqueued
-    cy.unmount()
+    // mount something else so that unmount is called
+    cy.mount(<div>Test Component</div>)
```

Comment:
Should we use the suggested alternative:
cy.then(() => ReactDom.unmountComponentAtNode(getContainerEl()))